aiida_wannier90/orbitals.py

```diff
@@ -31,18 +31,23 @@ def _generate_wannier_orbitals( # pylint: disable=too-many-arguments,too-many-lo
     can be provided either in cartesian coordiantes using position_cart
     or can be assigned based on an input structure and kind_name.
 
+    ..note:: cccc
+      xx
+      xx
+
+
     :param position_cart: position in cartesian coordinates or list of
                           positions in cartesian coodriantes
     :param structure: input structure for use with kind_names
     :param kind_name: kind_name, for use with the structure
     :param radial: number of radial nodes
     :param ang_mtm_name: orbital name or list of orbital names, cannot
-                         be used in conjunction with ang_mtm_l or
-                         ang_mtm_mr
-    :param ang_mtm_l: angular momentum, if ang_mtm_mr is not specified
-                      will return all orbitals associated with it
-    :param ang_mtm_mr: magnetic angular momentum number must be specified
-                       along with ang_mtm_l
+                         be used in conjunction with ang_mtm_l_list or
+                         ang_mtm_mr_list
+    :param ang_mtm_l_list: angular momentum (either an integer or a list), if 
+                 ang_mtm_mr_list is not specified will return all orbitals associated with it
+    :param ang_mtm_mr_list: magnetic angular momentum number must be specified
+                       along with ang_mtm_l_list
```

Comment:
Maybe say that if this is specified, `ang_mtm_l_list` must be an integer and not a list